packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -57,6 +105,29 @@ export default function useFeedSettings(): FeedSettingsReturnType {
     }, 100);
   }, [tagsCategories, feedSettings, avoidRefresh]);
 
+  useEffect(() => {
+    const isEmpty = isObjectEmpty(feedSettings);
+    const isFeedQueryEmpty = Object.keys(feedQuery).length === 0;
```

Comment:
Can you not use `isObjectEmpty` for the isFeedQueryEmpty as well?

Reply:
That makes sense, it will make the condition future proof even though we declare a default value of object literal in `feedQuery`. Let me refactor it.